modules/Cargo.toml

```diff
@@ -50,18 +50,18 @@ tokio = { version = "1.4", features = ["full"] }
 
 ## crates in order of publishing ## see docs/Publishing.md
 
-abstract-interface = { version = "0.21.0" }
-abstract-adapter = { version = "0.21.0" }
-abstract-app = { version = "0.21.0" }
+abstract-interface = { version = "0.21.0", default-features = false }
+abstract-adapter = { version = "0.21.0", default-features = false }
+abstract-app = { version = "0.21.0", default-features = false }
 abstract-testing = { version = "0.21.0" }
 abstract-macros = { version = "0.21.0" }
 abstract-client = { version = "0.21.0" }
 
 abstract-adapter-utils = { version = "0.21.0" }
-abstract-dex-standard = { version = "0.21.0" }
-abstract-staking-standard = { version = "0.21.0" }
+abstract-dex-standard = { version = "0.21.0", default-features = false }
+abstract-staking-standard = { version = "0.21.0", default-features = false }
```

Comment:
need to disable default features for everything inside workspace so contracts have control over features